client/src/pages/FiltersPage/FiltersPage.css

```diff
@@ -72,7 +72,6 @@
   background-color: white;
   height: 90vh;
   border-right: 1px solid #e4e4e4;
-  overflow-y: scroll;
```

Comment:
this will make it impossible to scroll in the list view so keep the `overflow-y: scroll`. you'll have to do some css scavenging to figure out what's causing the overflow